scripts/replaceStylableImports.js

```
@@ -20,11 +20,54 @@ module.exports = () => {
             reject();
             return;
           }
-          const codeWithEsImport = content.replace(
-            /wix-ui-core\/index\.st\.css/g,
-            'wix-ui-core/index.es.st.css',
+
+          let results = content; //eslint-disable-line
+
+          const hocs = /wix-ui-core\/dist\/src\/hocs\/.*\/([A-Za-z]*)\.st\.css/;
+
+          const foundStylesheets = results
+            .split('\n')
+            .filter(row => row.match(hocs))
+            .map(row => row.match(hocs)[1]);
+
+          results = results
+            .replace(
+              /wix-ui-core\/dist\/src\/hocs\/.*\/[A-Za-z]*\.st\.css/g,
+              'wix-ui-core/hocs.es.st.css',
+            )
+            .replace(
+              /wix-ui-core\/index\.st\.css/g,
+              'wix-ui-core/index.es.st.css',
+            )
+            .replace(
+              /wix-ui-core\/hocs\.st\.css/g,
+              'wix-ui-core/hocs.es.st.css',
+            );
+
+          foundStylesheets.forEach(
```

as discussed - we can just throw an error during build time if we see hoc imports from dist the same way as with components.